src/main/external-resources/renderers/DefaultRenderer.conf

```diff
@@ -566,14 +566,14 @@ CharMap =
 #    3gp      3rd Generation Partnership Project            video/3gpp
 #    3g2a     3rd Generation Partnership Project 2 Audio    audio/3gpp2
 #    3g2      3rd Generation Partnership Project 2          video/3gpp2
-#    aac-lc   Advanced Audio Coding (Low Complexity)        audio/aac
+#    aac-lc   Advanced Audio Coding (Low Complexity)        audio/mp4
```

Comment:
Done.
I know that i leaved the one for ALAC, but as i remember that yourself seem to have some issue with it, at least we don't understand each other on that, so i thought that it could help users more than confuse them (I wished to add it to the MP4 lines but it is already full)